### PR TITLE
[plugin.video.ring_doorbell] 1.0.1

### PR DESCRIPTION
**Pull request by jlippold**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.ring_doorbell/addon.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.ring_doorbell" name="Ring video doorbell" version="1.0.1" provider-name="Jed Lippold, tchellomello">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0" />
```
Comment (enen92) [inline plugin.video.ring_doorbell/addon.xml]: 2.25.0 please to match Krypton

File: plugin.video.ring_doorbell/addon.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <import addon="script.module.requests" version="2.12.4" />
+        <import addon="script.module.dateutil" version="2.5.3" />
+        <import addon="script.module.pytz" version="2014.2" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <license>GNU General Public License, v2</license>
+        <description lang="en">Play ring doorbell saved recording from Kodi. Requires a paid Ring.com subscription</description>
```
Comment (enen92) [inline plugin.video.ring_doorbell/addon.xml]: description lang attribute should be en_GB (new default for krypton)

**Comment by jlippold** [comment]
all set @enen92

File: plugin.video.ring_doorbell/ring_doorbell/const.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+           'Accept-Encoding': 'gzip, deflate'}
+
+# number of attempts to refresh token
+RETRY_TOKEN = 3
+
+# default suffix for session cache file
+CACHE_ATTRS = {'account': None, 'alerts': None, 'token': None}
+
+try:
+    CACHE_FILE = os.path.join(os.getenv("HOME"),
```
Comment (enen92) [inline plugin.video.ring_doorbell/ring_doorbell/const.py]: I am afraid you can only write to the userdata of your addon. Can you please adapt?

**Comment by enen92** [comment]
Also your fanart does not seem to have the correct dimensions. See here:

http://kodi.wiki/view/Add-on_rules

Sorry for not pointing this out on the first "review"

**Comment by enen92** [comment]
ping @jlippold

**Comment by jlippold** [comment]
ok @enen92 all set

**Comment by enen92** [comment]
Thanks, merged

**Comment by MartijnKaijser** [comment]
@jlippold could you fix this?
PROBLEM: Icon should have either 256x256 or 512x512 but it has 310x310

**Pull request merged**